### PR TITLE
AWS-4497: Update readme to reflect end of updates to gem

### PR DESCRIPTION
**Pull request by NatahlieWang**
Updating README to reflect we are not going to be updating the gem for amazon-pricing any further

Before:
Screen Shot 2019-03-13 at 3 29 44 PM

After:
Screen Shot 2019-08-01 at 10 50 46 AM

### CODE REVIEW

**Comment by storpeyCHT** [comment]
I was unable to detect any tests in your PR.  Please fill out this survey to help make testing easier https://goo.gl/forms/B4Onm8mKfw7n9PB33

File: README.md [hunk 1]
```diff
@@ -1,6 +1,18 @@
...
+DEPRECATED: Amazon Web Services Pricing Ruby Gem
 ====================================
 
+⛔️ NOTICE OF FUTURE UPDATES TO THE GEM ⛔️
+====================================
+This project is no longer being maintained and will be deprecated in the future. Issues and pull requests will no longer be accepted. If you have scripts that utilize this project, consider using alternate libraries.
+
+Amazon has updated the method it uses to provide pricing information to be an API; as such the methods contained within this Gem are no longer useful for obtaining pricing information.
+
+Updates to the gem will be few and far between; please contact *nbeavers@vmware.com* with any questions.
```
Comment (paul-lothridge) [inline README.md]: shouldn't direct this to a general mailbox, but specific to pricing, i.e. not a personal address?

File: lib/amazon-pricing/version.rb [hunk 2]
```diff
@@ -3,12 +3,12 @@
...
 # Author::    Joe Kinsella (mailto:joe.kinsella@gmail.com)
-# Copyright:: Copyright (c) 2011-2018 CloudHealth
+# Copyright:: Copyright (c) 2011-2019 CloudHealth
 # License::   Distributes under the same terms as Ruby
 # Home::      http://github.com/CloudHealth/amazon-pricing
 #++
 module AwsPricing
 
-VERSION = '0.1.135' # [major,minor.fix]: Moving instance-type methods to be class methods rather than instance methods
+VERSION = '2.1.000' # [major,minor.fix]: MAJOR: Gem is no longer going to be actively supported or maintained. Information in readme; reachout to nbeavers@vmware.com for more information.
```
Comment (paul-lothridge) [inline lib/amazon-pricing/version.rb]: same comment as above, re: personal email address

File: lib/amazon-pricing/version.rb [hunk 3]
```diff
@@ -3,12 +3,12 @@
...
 # Author::    Joe Kinsella (mailto:joe.kinsella@gmail.com)
-# Copyright:: Copyright (c) 2011-2018 CloudHealth
+# Copyright:: Copyright (c) 2011-2019 CloudHealth
 # License::   Distributes under the same terms as Ruby
 # Home::      http://github.com/CloudHealth/amazon-pricing
 #++
 module AwsPricing
 
-VERSION = '0.1.135' # [major,minor.fix]: Moving instance-type methods to be class methods rather than instance methods
+VERSION = '2.1.000' # [major,minor.fix]: MAJOR: Gem is no longer going to be actively supported or maintained. Information in readme; reachout to cht-aws-pricing-services@groups.vmware.com for more information.
```
Comment (stanmluke) [inline lib/amazon-pricing/version.rb]: the main issue when I attempted bumping (only the minor) was that all the other CHT repos 'broke' (i.e. overly restrictive gem version in Gemfile);  hence, we should test the blast radius on bumping this.

**Pull request merged**